public/icons/1840/yellow_hex.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32"  stroke="#000" fill="#fde900" xmlns:v="https://vecta.io/nano"><path stroke-width="2" d="M2.5 13.049L16 3.5l13.5 9.549L24.343 28.5H7.657L2.5 13.049z"/><path d="M1 291.764l149-.382 149 .382-56.913.618H57.913L1 291.764z"/></svg>
```

Comment:
`stroke="#000"`
… is default. Remove.

`xmlns:v="https://vecta.io/nano"`
Remove.
(… from all svg files)